docs/modules/wan/pages/tuning.adoc

```diff
@@ -612,10 +612,12 @@ ordering, when these entries are added, is not necessary.
 Keep in mind that if you set this element to a value which is less than the target endpoint count,
 you will lose performance as not all target endpoints will be used at any point in time to process the WAN event batches.
 So, for instance, if you have a target cluster with 3 members (target endpoints) and you want to use
-this element, it only makes sense to set it to a value higher than 3. Otherwise, you can simply disable it by
+this element, it makes sense to set it to a value equal to or greater than 3. Otherwise, you can simply disable it by
 setting it to less than 2 in which case WAN will use the default replication strategy and adapt to
 the target endpoint count while maintaining causality.
 
+NOTE: Configuring the `max-concurrent-invocations` above 1 will improve WAN replication speed, but this will be a trade-off for higher CPU consumption as more threads are involved in WAN replication. If your replication setup is not saturating WAN queues (so there is no demand for faster WAN replication) then this functionality should not be used as serial replication will be sufficient without additional overhead or out-of-order risks.
```

Comment:
```suggestion
NOTE: Configuring the `max-concurrent-invocations` to a value greater than 1 will improve WAN replication speed, but this will be a trade-off for higher CPU consumption as more threads are involved in WAN replication. If your replication setup is not saturating WAN queues (so there is no demand for faster WAN replication) then this functionality should not be used as serial replication will be sufficient without additional overhead or out-of-order risks.
```

Reply:
I read above and initially thought it was referring to information "above" - I think it is clearer with my suggestion, but I'll leave it to your discretion (it could be the result of having read the note without the context)

Reply:
I agree, your suggestion makes the text clearer - thank you!